{"name": "针对异常登陆的测试", "status": "passed", "steps": [{"name": "步骤1：登录", "status": "passed", "start": 1609923348958, "stop": 1609923348960}, {"name": "开始断言", "status": "passed", "start": 1609923350038, "stop": 1609923350039}], "attachments": [{"name": "log", "source": "0f0a95fb-d8d9-4bf5-b554-bc50ba6f9a66-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录功能-异常登录-用户名错误', 'username': 'abc', 'password': '654321Ok'}"}], "start": 1609923348955, "stop": 1609923350951, "uuid": "306cc041-afc6-4c51-a055-d302637899b4", "historyId": "28fbee3f7849e5c692210a9f12e7fe96", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "epic", "value": "金融界web自动化自测"}, {"name": "feature", "value": "金融界web登录模块"}, {"name": "story", "value": "登录模块-异常登陆"}, {"name": "tag", "value": "@pytest.mark.usefixtures('refresh_page')"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "PC-20200718ELAI"}, {"name": "thread", "value": "55540-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}